docs/source/getting_involved.rst

```diff
@@ -19,18 +19,11 @@ Getting involved
 Cartographer is developed in the open and allows anyone to contribute to the project.
 There are multiple ways to get involved!
 
-Twice a month, the project hosts "Open House Hangouts" sessions that are essentially meetings open to everyone to join in.
-The call typically recaps the recent and ongoing development around Cartographer and Cartographer ROS.
-The developers are then open to questions from the community, this is a great time to ask about contribution ideas.
-If you don't feel like talking or being seen, you are free to join anyway and skulk!
-The slides are also made available after each session but there is no video recording.
-
-If you want to stay tuned with announcements (such as new major releases or new open house sessions), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
+If you want to stay tuned with announcements (such as new major releases), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
```

Comment:
@wohe @bochen87 I would keep it like this. Or did you have other plans for the mailing list that I don't know?

Reply:
no, we didn't - maybe to revive more active involvement from the community, we could do some more regular sessions? like monthly or bi-monthly?

Reply:
Regarding the mailing list: I do not think anyone of us has admin access to the old group and it does not seem there has been anything happening there in the last years. Maybe we give it up and point people towards GitHub issues where they actually have a chance to get some info/answers?

Reply:
yes, I think this makes the most sense.